packages/test/amplify_integration_test/README.md

```diff
@@ -0,0 +1,3 @@
+# amplify_integration_test
+
+Utilities for running Amplify integration tests.
```

Comment:
Can we give more details here, at minimum can you include the details regarding package dependencies mentioned in your Github comment?  I'm concerned we might loose track of that insight.